packages/utils/package.json

```diff
@@ -48,16 +48,6 @@
             "types": "./dist/types/f1-field.d.ts",
             "require": "./dist/lib.commonjs/f1-field.cjs",
             "default": "./dist/lib.esm/f1-field.js"
-        },
-        "./crypto": {
```

Comment:
why are we removing this?

Reply:
The Rollup conf with `preserveModules: true`, which is used to compile the ts files to js without bundling in 1 file, doesn't include the `crypto/crypto.browser.ts` file, because it is not actually imported by `index.ts`. I haven't found any simple solution so far, so I opted not to use that conditional export for now. `crypto` can be imported from `@zk-kit/utils` only for new.